v3/how-to-guides/08-tools/d-subwasm/index.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Verify Runtime Wasm Binaries with Subwasm
+slug: /how-to-guides/v3/tools/subwasm
+keywords: tooling, wasm
+version: 3.0 
+section: how to guides
+category: tools
+---
+
+[Subwasm](https://github.com/chevdor/subwasm) provides a way to expose the features of the runtime for any Substrate chain. 
+This guide steps through the different capabilities that Subwasm provides. 
+
+## Use cases
+
+- Expose verifiable historical information of a chain's runtime for front end applications.
+- Get a quick snapshot of the modules in a runtime.
+- Check for changes between runtimes using a diff.
+- Expose a verifiable runtime for governance transparency.
+
+## Before you begin
+
+To follow this guide you should have a runtime from a chain you wish to query. 
+This could either be one running locally or one running in production.
+In order to filter metadata, you will also need to [install JQ](https://stedolan.github.io/jq/download/).
+
+## Install and discover the CLI tool
+
+To get started, install Subwasm and explore its commands.
+
+```bash
+cargo install --locked --git https://github.com/chevdor/subwasm --tag v0.16.1
+```
+Make sure you're using the [latest release](https://github.com/chevdor/subwasm/releases). 
+This guide is based on version 0.14.1. 
+
+Explore the CLI commands.
+
+```bash
+subwasm --help
+```
+
+## Get information about a runtime 
+
+1. Launch a local node from inside your chain's directory. 
+   This example is launching the node template in dev mode.
+   
+    ```bash
+    ./target/release/node-template --tmp --dev
```

Comment:
Can you do this for any pubclic node of any chain? What flags are needed for the node to allow for this (if any)?
Let's clarify and add an example with some public node too :pray:

Reply:
Made it clearer that then first part uses subwasm for a node running locally and the second uses it for a live chain.